language-design.md

```diff
@@ -102,7 +243,31 @@ you do so? If not, why not?
 
 **Response**
 
-
+These experiences of natural languages are not at odds with one another. 
+Pavlus only mentioned "natural-language replacements for some of the more 
+abstruse syntax". Perhaps what happened with AppleScript was too much similarity 
+with natural languages. 
+
+The study Pavlus talked about referred to the language Quorum, which seems like 
+a very similar language to Python. Python is also a language that is seen as 
+very beginner friendly. Many of its syntax is, like Quorum, very "natural" to 
+someone who has not programmed before, for example, "if number in numbers." 
+However, Python would never be mistaken for any natural language in even a 
+brief inspection - neither would Quorum. Once a language is too much like 
+natural languages, the user might use their instincts in the natural languages 
```

Comment:
I'm curious what you two mean by "instincts in the natural language." I read the quote to mean because AppleScript was so close to human language, people tried to express themselves too many different ways because they thought AppleScript actually understood English, instead of just understanding certain commands. Whether or not we read it in the same way, I find it interesting that your team call them "instincts" because I think a lot of instincts about how to solve problems are the basis for solving most bugs. This is why grutors often ask students to explain a bug in English or explain what they want their code to do.